README.md

```diff
@@ -2,24 +2,26 @@
 [![Stories in Ready](https://badge.waffle.io/ProjectMirador/mirador.svg?label=ready&title=Ready)](http://waffle.io/iiif/mirador)
 
 # Mirador
-![mirador banner](http://projectmirador.github.io/mirador/img/banner.jpg)
+![mirador banner](https://projectmirador.github.io/mirador/img/banner.jpg)
 **Mirador is a multi-repository, configurable, extensible, and easy-to-integrate viewer and annotation creation and comparison environment for IIIF resources, ranging from deep-zooming artwork, to complex manuscript objects. It provides a tiling windowed environment for comparing multiple image-based resources, synchronised structural and visual navigation of content using openSeadragon, Open Annotation compliant annotation creation and viewing on deep-zoomable canvases, metadata display, bookreading, and bookmarking.**
 
 ### [See a Demo](http://projectmirador.org/demo/)
 ### [Getting Started](http://projectmirador.org/docs/docs/getting-started.html)
 
 ### Run in Development
-Mirador uses [node.js](http://nodejs.org/) and a build system to assemble, test, and manage the development resources. If you have never used these tools before, you may need to install them.
+Mirador uses [Node.js](https://nodejs.org/) and a build system to assemble, test, and manage the development resources. If you have never used these tools before, you may need to install them.
 
- 1. Install Node, if you haven't already (available at the link above)  
- 1. Clone the mirador repository (if you haven't already done so above)
- 1. On the command line, go into the mirador folder
- 1. Install all dependencies with `npm install`. Run `npm start'.
+ 1. Install [Node.js](https://nodejs.org/)
+ 2. Install the Grunt command line runner i.e. `npm install -g grunt-cli`
+ 3. Install the Bower command line utility i.e. `npm install -g bower`
```

Comment:
I'm not sure this is necessary anymore. Perhaps we need to update the documentation elsewhere?

Reply:
Maybe? I didn't test it in its entirety. I had trouble at first, but it might have only been due to not installing `grunt`. Also `grunt` itself shows various errors, but `grunt serve` works fine regardless.

Reply:
Looks like it might still be :( . Hopefully we can change that.

Reply:
https://github.com/ProjectMirador/mirador/blob/9e0c99e69418af84b375a481973ae8f4a7042154/Gruntfile.js#L48

Reply:
Also, `grunt test` does not seem to work. Unless I am misunderstanding something, should it be removed?

Reply:
I'm not seeing a `grunt test` task. Is `npm test` not working for you?

Reply:
Oh! I meant here: http://projectmirador.org/docs/

Reply:
Gotcha. Yep, I'm not sure about the validity of those docs, but most definitely `grunt test` is gone.

Reply:
Is there a way to update them?